examples/ssh/opkssh/cli_test.go

```diff
@@ -18,24 +21,30 @@ func AllowAllPolicyEnforcer(userDesired string, pkt *pktoken.PKToken) error {
 }
 
 func TestAuthorizedKeysCommand(t *testing.T) {
-	op, err := providers.NewMockOpenIdProvider()
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	principals := []string{"guest", "dev"}
 	alg := jwa.ES256
-
 	signer, err := util.GenKeyPair(alg)
 	if err != nil {
 		t.Fatal(err)
 	}
-	email := "arthur.aardvark@example.com"
-	pkt, err := mocks.GenerateMockPKTokenWithEmail(signer, alg, email)
+
+	// extra ID token payload claims
+	mockEmail := "arthur.aardvark@example.com"
+	extraClaims := map[string]any{
+		"email": mockEmail,
+	}
+
+	op, err := providermocks.NewMockOpenIdProvider(t, extraClaims)
 	if err != nil {
 		t.Fatal(err)
 	}
```

Comment:
Not a issue you introduced, but since you are editing the code on either side, it is worth fixing

```suggestion
require.NoError(t, err)
```